db/samplev2.py

```diff
@@ -60,14 +60,18 @@ def generate_sample_db(path, course_id, course_name, db_session):
             db_session.rollback()
 
         # Add scores of the student
+        #If you want the entries to be displayed in order in DB Browser then use this otherwise comment it.
+        if(db_session.query(Score).filter_by(student_id = formatted_student_id).first() != None):
+            db_session.query(Score).filter_by(student_id = formatted_student_id, course_id = course_id).delete(synchronize_session = 'fetch')
```

Comment:
Why is this `synchronize_session = 'fetch'` needed?

Reply:
The 'fetch' strategy results in an additional SELECT statement emitted and will significantly reduce performance.

Reply:
False - don’t synchronize the session. This option is the most efficient and is reliable once the session is expired, which typically occurs after a commit(), or explicitly using expire_all(). Before the expiration, objects may still remain in the session which were in fact deleted which can lead to confusing results if they are accessed via get() or already loaded collections.

'fetch' - performs a select query before the delete to find objects that are matched by the delete query and need to be removed from the session. Matched objects are removed from the session.

I figured fetch was the correct option for our purpose..

Reply:
Actually evaluate will also serve the purpose.. Ill update it